sdk/examples/wallet/spammer.rs

```diff
@@ -0,0 +1,229 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! In this example we will spam transactions from multiple threads simultaneously to our own addresses.
+//!
+//! Rename `.env.example` to `.env` first, then run the command:
+//! ```sh
+//! cargo run --release --features=wallet --example spammer
+//! ```
+
+use std::{env::var, time::Duration};
+
+use iota_sdk::{
+    client::{
+        constants::SHIMMER_COIN_TYPE,
+        request_funds_from_faucet,
+        secret::{mnemonic::MnemonicSecretManager, SecretManager},
+    },
+    types::block::{address::Bech32Address, payload::transaction::TransactionId},
+    wallet::{account::FilterOptions, Account, ClientOptions, Result, SendAmountParams, Wallet},
+};
+use tokio::{
+    task::JoinSet,
+    time::{sleep, Instant},
+};
+
+// The account alias used in this example.
+const ACCOUNT_ALIAS: &str = "spammer";
+// The number of spamming rounds.
+const NUM_ROUNDS: usize = 1000;
+// The amount to send in each transaction
+const SEND_AMOUNT: u64 = 1_000_000;
+// The number of simultaneous transactions
+const NUM_SIMULTANEOUS_TXS: usize = 64;
+
+#[tokio::main(flavor = "multi_thread")]
+async fn main() -> Result<()> {
+    // This example uses secrets in environment variables for simplicity which should not be done in production.
+    dotenvy::dotenv().ok();
+
+    let num_simultaneous_txs = NUM_SIMULTANEOUS_TXS.min(num_cpus::get());
+
+    println!("Spammer set up to issue {num_simultaneous_txs} transactions simultaneously.");
+
+    // Restore wallet from a mnemonic phrase.
```

Comment:
Not a restore?

Reply:
Depends how we define "restore" I would say. Since we don't go through stronghold here (get access to the stored seed via a password), but instead provide the mnemonic/seed directly you could call it a restore. If you lose access to your stronghold snapshot, but still have the mnemonic this is exactly what you'ld have to do to "restore" your wallet.

Reply:
You could also just store the mnemonic in a new stronghold and use that, I also think the name restore here sounds a bit wrong